assignment4/ques3.js

```diff
@@ -0,0 +1,21 @@
+// Write a program to give count of variables in a string ("abcabcdabbcc" => "a3b4c4d1")
+let inputString = "abcabcdabbcc";
+
+let countMap = new Map();
+
+for (let i = 0; i < inputString.length - 1; i++) {
```

Comment:
Your loop iterates until inputString.length - 1, this might stops short of the last character in the string. You should iterate until inputString.length. please check output of the program once and do the needful changes